src/instructlab/model/serve.py

```diff
@@ -94,6 +94,14 @@ def signal_handler(
         "Automatically detected based on the model file properties.\n"
     ),
 )
+@click.option(
+    "--gpus",
```

Comment:
so it's `--gpus` in CLI but `tensor_parallel_size` in config? This is confusing. What's the rationale to have different names for these?